R/sample.R

```diff
@@ -170,9 +171,14 @@ print.step_sample <-
 #' @rdname tidy.recipe
 #' @export
 tidy.step_sample <- function(x, ...) {
-  tibble(
-    size = x$size,
-    replace = x$replace,
-    id = x$inputs
-  )
+  if (is.null(x$size)) {
+    res <- tibble(size = numeric(), replace = logical())
+  } else {
+    res <- tibble(
+      size = x$size,
+      replace = x$replace
+    )
+  }
+  res$id <- x$id
```

Comment:
It looks like this may have previously referenced `x$inputs` rather than `x$id`?

Reply:
yes, and it shouldn't have. `$inputs` doesn't exists, and will return `NULL`